packages/ui-contract-editor/src/lib/components/Conditional/index.js

```diff
@@ -81,11 +81,11 @@ const Conditional = React.forwardRef((props, ref) => {
   };
 
   return (
-    <>
+    <span {...attributes}>
         { isNotReadOnly && isNotEmptyString && <ConditionalSwitch {...conditionalSwitchProps} /> }
         { isNotReadOnly && isEmptyString && <ConditionalBoolean {...conditionalIconProps} /> }
         <span {...conditionalProps}>{children}</span>
-    </>
+    </span>
```

Comment:
See note on other comment. It should be passed as a prop to the React component and spread here. Otherwise, when spread on the React component itself, we are just passing each individual property of attributes as a prop to the component and then not doing anything with it.